pkgs/applications/virtualization/looking-glass-client/default.nix

```diff
@@ -2,6 +2,7 @@
 , freefont_ttf, spice-protocol, nettle, libbfd, fontconfig, libffi, expat
 , libxkbcommon, libGL, libXext, libXrandr, libXi, libXScrnSaver, libXinerama
 , libXcursor, libXpresent, wayland, wayland-protocols
+, terminal ? true
```

Comment:
With this name I find it risky.  When someone adds a top-level `terminal` package, it will get auto-passed here instead of `true`.